<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>571</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">.cFlatOptionButton:!checked
{
	background-color:  palette(dark);
}
.cFlatOptionButton:!checked:pressed
{
	background-color:  palette(shadow);
}
.cFlatOptionButton:checked
{
	background-color:  palette(highlight);
	color: palette(highlighted-text);
}
.cFlatOptionButton:checked:pressed
{
	background-color:  palette(shadow);
	color: palette(button-text);
}
.cFlatOptionButton:disabled
{
	background-color: palette(dark);
	color: palette(mid);
}
.cFlatOptionButton
{
	padding: 1px 4px 1px 4px;
	margin: 0px;
	spacing: 0px;
	border-radius: 1px;
}
#mMainContainerTop
{
	background-color:palette(mid);
}

#mViewEditImagePreviewInput, #mViewEditImagePreviewOutput
{
    background: palette(base);
}
#mViewEditImagePreviewInput  QScrollBar, #mViewEditImagePreviewOutput  QScrollBar
{
    border: 0;
    background: palette(base);
    margin: 0;
	padding: 0;
}
#mViewEditImagePreviewInput  QScrollBar::handle, #mViewEditImagePreviewOutput  QScrollBar::handle
{
    border-radius:1px;
}
#mViewEditImagePreviewInput QScrollBar:horizontal, #mViewEditImagePreviewOutput  QScrollBar:horizontal
{
    height: 5px;
}
#mViewEditImagePreviewInput  QScrollBar:vertical, #mViewEditImagePreviewOutput  QScrollBar:vertical
{
    width: 5px;
}
#mViewEditImagePreviewInput  QScrollBar::handle:horizontal, #mViewEditImagePreviewOutput  QScrollBar::handle:horizontal
{
    background: palette(highlight);
	min-width:20px;
}
#mViewEditImagePreviewInput  QScrollBar::handle:vertical, #mViewEditImagePreviewOutput  QScrollBar::handle:vertical
{
    background: palette(highlight);
	min-height:20px;
}
#mViewEditImagePreviewInput QScrollBar::add-line:horizontal, #mViewEditImagePreviewOutput QScrollBar::add-line:horizontal,
#mViewEditImagePreviewInput QScrollBar::add-line:vertical, #mViewEditImagePreviewOutput QScrollBar::add-line:vertical,
#mViewEditImagePreviewInput QScrollBar::sub-line:horizontal, #mViewEditImagePreviewOutput QScrollBar::sub-line:horizontal,
#mViewEditImagePreviewInput QScrollBar::sub-line:vertical, #mViewEditImagePreviewOutput QScrollBar::sub-line:vertical
{
	width:0;
}

#mViewEditProgressBarProcessingOutput
{
	border : 0px;
	background-color : #ed1461;
}
#mViewEditProgressBarProcessingOutput::chunk
{
	background-color : #00ffd8;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="mMainContainer" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="mMainContainerTop" native="true">
        <layout class="QHBoxLayout" name="mMainLayoutTop" stretch="0,1,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QStackedWidget" name="mMainContainerToolbars">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="mToolbarEdit">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="anitools::widgets::ToolButtonEx" name="mToolbarEditButtonLoadImage" native="true">
               <property name="toolTip">
                <string>Load image.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="mToolbarEditButtonSaveImage">
               <property name="toolTip">
                <string>Save image.</string>
               </property>
               <property name="icon">
                <iconset resource="imagebatchprocessor.qrc">
                 <normaloff>:/imagebatchprocessor/icons/saveImage</normaloff>
                 <disabledoff>:/imagebatchprocessor/icons/saveImageDisabled</disabledoff>:/imagebatchprocessor/icons/saveImage</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonIconOnly</enum>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="mToolbarEditSeparator01">
               <property name="minimumSize">
                <size>
                 <width>2</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>2</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="anitools::widgets::ToolButtonEx" name="mToolbarEditButtonAddFilter" native="true">
               <property name="toolTip">
                <string>Add image filter.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="anitools::widgets::ToolButtonEx" name="mToolbarEditButtonLoadFilters" native="true">
               <property name="toolTip">
                <string>Load image filter list.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="mToolbarEditButtonSaveFilters">
               <property name="toolTip">
                <string>Save image filter list.</string>
               </property>
               <property name="icon">
                <iconset resource="imagebatchprocessor.qrc">
                 <normaloff>:/imagebatchprocessor/icons/saveImageFilters</normaloff>
                 <disabledoff>:/imagebatchprocessor/icons/saveImageFiltersDisabled</disabledoff>:/imagebatchprocessor/icons/saveImageFilters</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonIconOnly</enum>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="mToolbarEditSeparator02">
               <property name="minimumSize">
                <size>
                 <width>2</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>2</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="mToolbarEditButtonVSplitter">
               <property name="toolTip">
                <string>Lay out preview vertically.</string>
               </property>
               <property name="icon">
                <iconset resource="imagebatchprocessor.qrc">
                 <normaloff>:/imagebatchprocessor/icons/vSplitter</normaloff>:/imagebatchprocessor/icons/vSplitter</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonIconOnly</enum>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">mButtonGroupToolbarEditSplitter</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="mToolbarEditButtonHSplitter">
               <property name="toolTip">
                <string>Lay out preview horizontally.</string>
               </property>
               <property name="icon">
                <iconset resource="imagebatchprocessor.qrc">
                 <normaloff>:/imagebatchprocessor/icons/hSplitter</normaloff>:/imagebatchprocessor/icons/hSplitter</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonIconOnly</enum>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">mButtonGroupToolbarEditSplitter</string>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="mToolbarBatch">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="mToolbarMoveHandle" native="true"/>
         </item>
         <item>
          <widget class="QWidget" name="mMainContainerChangeSection" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QToolButton" name="mMainButtonEdit">
              <property name="toolTip">
               <string>Switch to edit view.</string>
              </property>
              <property name="text">
               <string>Edit</string>
              </property>
              <property name="icon">
               <iconset resource="imagebatchprocessor.qrc">
                <normaloff>:/imagebatchprocessor/icons/edit</normaloff>:/imagebatchprocessor/icons/edit</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextBesideIcon</enum>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mButtonGroupMainChangeSection</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="mMainButtonBatch">
              <property name="toolTip">
               <string>Switch to batch processing view.</string>
              </property>
              <property name="text">
               <string>Batch Processing</string>
              </property>
              <property name="icon">
               <iconset resource="imagebatchprocessor.qrc">
                <normaloff>:/imagebatchprocessor/icons/batchProcessing</normaloff>:/imagebatchprocessor/icons/batchProcessing</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextBesideIcon</enum>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mButtonGroupMainChangeSection</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="mMainContainerViews">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="mViewEdit">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSplitter" name="mViewEditSplitterMain">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="handleWidth">
             <number>1</number>
            </property>
            <property name="childrenCollapsible">
             <bool>false</bool>
            </property>
            <widget class="QWidget" name="verticalLayoutWidget_3">
             <layout class="QVBoxLayout" name="mViewEditLayoutPreview">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QSplitter" name="mViewEditSplitterPreview">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="handleWidth">
                 <number>1</number>
                </property>
                <property name="childrenCollapsible">
                 <bool>false</bool>
                </property>
                <widget class="QWidget" name="verticalLayoutWidget">
                 <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1">
                  <property name="spacing">
                   <number>5</number>
                  </property>
                  <item>
                   <widget class="anitools::widgets::ImageViewer" name="mViewEditImagePreviewInput" native="true">
                    <widget class="QWidget" name="mViewEditContainerInputZoom" native="true">
                     <property name="geometry">
                      <rect>
                       <x>400</x>
                       <y>210</y>
                       <width>181</width>
                       <height>21</height>
                      </rect>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout">
                      <property name="spacing">
                       <number>5</number>
                      </property>
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <item>
                       <widget class="QSlider" name="mViewEditSliderInputZoom">
                        <property name="toolTip">
                         <string>Change the zoom factor of the input image preview.</string>
                        </property>
                        <property name="maximum">
                         <number>13</number>
                        </property>
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QComboBox" name="mViewEditComboInputZoom">
                        <property name="toolTip">
                         <string>Change the zoom factor of the input image preview.</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="verticalLayoutWidget_2">
                 <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1">
                  <property name="spacing">
                   <number>5</number>
                  </property>
                  <item>
                   <widget class="anitools::widgets::ImageViewer" name="mViewEditImagePreviewOutput" native="true">
                    <widget class="QProgressBar" name="mViewEditProgressBarProcessingOutput">
                     <property name="geometry">
                      <rect>
                       <x>170</x>
                       <y>50</y>
                       <width>221</width>
                       <height>5</height>
                      </rect>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>5</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>5</height>
                      </size>
                     </property>
                     <property name="textVisible">
                      <bool>false</bool>
                     </property>
                    </widget>
                    <widget class="QWidget" name="mViewEditContainerOutputZoom" native="true">
                     <property name="geometry">
                      <rect>
                       <x>400</x>
                       <y>190</y>
                       <width>181</width>
                       <height>21</height>
                      </rect>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_5">
                      <property name="spacing">
                       <number>5</number>
                      </property>
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <item>
                       <widget class="QSlider" name="mViewEditSliderOutputZoom">
                        <property name="toolTip">
                         <string>Change the zoom factor of the output image preview.</string>
                        </property>
                        <property name="maximum">
                         <number>13</number>
                        </property>
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QComboBox" name="mViewEditComboOutputZoom">
                        <property name="toolTip">
                         <string>Change the zoom factor of the output image preview.</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="horizontalLayoutWidget_2">
             <layout class="QHBoxLayout" name="mViewEditLayoutScrollArea" stretch="1">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QScrollArea" name="mViewEditWidgetListScrollArea">
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="horizontalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="anitools::widgets::WidgetList" name="mViewEditWidgetList">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>764</width>
                   <height>519</height>
                  </rect>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="mViewBatch"/>
       </widget>
      </item>
     </layout>
     <zorder>mMainContainerViews</zorder>
     <zorder>mMainContainerTop</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>anitools::widgets::WidgetList</class>
   <extends>QWidget</extends>
   <header>widgetlist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>anitools::widgets::ToolButtonEx</class>
   <extends>QWidget</extends>
   <header>toolbuttonex.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>anitools::widgets::ImageViewer</class>
   <extends>QWidget</extends>
   <header>imageviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="imagebatchprocessor.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="mButtonGroupMainChangeSection"/>
  <buttongroup name="mButtonGroupToolbarEditSplitter"/>
 </buttongroups>
</ui>
